requirements.txt

```diff
@@ -20,6 +20,8 @@ ovh==0.5.0
 pycountry-convert==0.7.2
 pycountry==20.7.3
 python-dateutil==2.8.1
-requests==2.24.0
+requests==2.25.1
 s3transfer==0.3.3
 setuptools==44.1.1
+six==1.15.0
```

Comment:
Anything specifically requiring `six` here? If it's just pulled in as a dependency of a dependency it can be left out of `requirements.txt` as we aren't doing a full `freeze` for them.

Reply:
A result of the Revert, not needed indeed.